submit/templates/submit/base.html

```diff
@@ -18,6 +18,16 @@ <h1 class="title">
     <h2 class="replacement is-size-5 has-text-dark">Replacement for: arXiv:{{ submission.arxiv_id }}v{{ submission.version }} {{ submission.metadata.title }}</h2>
   {% endif %}
   <content>
```

Comment:
If electing to upgrade to base 0.14.3, please change `<content>` tag to `<main>` - `<content>` is a deprecated element and not valid HTML5 (so I learned recently - my bad on putting it in to begin with). You may also run into this as a merge conflict to develop, can't remember if I changed submit/base.html yet or not since this branch was opened.